os/arch/arm/src/imxrt/imxrt_ehci.h

```diff
@@ -0,0 +1,108 @@
+/************************************************************************************
+ * arch/arm/src/imxrt/imxrt_ehci.h
```

Comment:
1. correct the path:
arch/arm/src/imxrt/imxrt_ehci.h  --> os/arch/arm/src/imxrt/imxrt_ehci.h

2. Add Samsung License part as well.